lib/cli.mjs

```diff
@@ -468,8 +468,14 @@ export default async function main(argv) {
 		options.cacheStrategy = cacheStrategy;
 	}
 
-	if (isBoolean(fix)) {
-		options.fix = fix;
+	if (isString(fix)) {
+		if (fix === 'true' || fix === '') {
+			options.fix = true;
+		} else if (fix === 'false') {
+			options.fix = false;
+		} else if (fix === 'strict' || fix === 'lax') {
+			options.fix = fix;
+		}
```

Comment:
It seems that this test is not being reflected in the coverage.
https://github.com/stylelint/stylelint/pull/8106/files#diff-0104e2476d83a456a5e8add0acc8334030af22abad28e606b33f9d8094c04e5dR106-R110

Reply:
No, directly calling `buildCLI()` (in the test case) doesn't call `main()`. We should believe the coverage report and add a proper case.

Reply:
Thanks! I've added.
[test: add cli coverage for --fix options](https://github.com/stylelint/stylelint/pull/8106/commits/8c04faae57c115370a903816af588610ec529cdc)